Content.Shared/EntityEffects/Effects/EvenHealthChange.cs

```diff
@@ -91,10 +96,47 @@ public override void Effect(EntityEffectBaseArgs args)
         }
 
         var damagableSystem = args.EntityManager.System<DamageableSystem>();
-        var universalReagentDamageModifier = damagableSystem.UniversalReagentDamageModifier;
-        var universalReagentHealModifier = damagableSystem.UniversalReagentHealModifier;
+        var dspec = GetDamageSpec(protoMan, damageable); // DeltaV - Fix EvenHealing with Limbs.
+
+        damagableSystem.TryChangeDamage(
+            args.TargetEntity,
+            dspec * scale,
+            IgnoreResistances,
+            interruptsDoAfters: false,
+            doPartDamage: false);   // DeltaV - Fix EvenHealing with Limbs.
+
+        // DeltaV Start - Fix EvenHealing with Limbs.
+        var bodySystem = args.EntityManager.System<SharedBodySystem>();
+        var bodyParts = SharedTargetingSystem.GetValidParts();
+        foreach (var bodyPart in bodyParts)
+        {
+            var (targetType, targetSymmetry) =  bodySystem.ConvertTargetBodyPart(bodyPart);
+            if (bodySystem.GetBodyChildrenOfType(args.TargetEntity, targetType, symmetry: targetSymmetry) is { } part)
+            {
+                if (!args.EntityManager.TryGetComponent<DamageableComponent>(part.FirstOrDefault().Id, out var damageablePart))
+                    continue;
+                dspec = GetDamageSpec(protoMan, damageablePart);
+
+                if (dspec.GetTotal() == 0)
+                    continue;
+
+                damagableSystem.TryChangeDamage(
+                    args.TargetEntity,
+                    dspec * scale,
+                    IgnoreResistances,
+                    interruptsDoAfters: false,
+                    targetPart: bodyPart,
+                    onlyDamageParts: true,
+                    partMultiplier: 0.5f,
+                    canSever: false);
+            }
+        }
+        // DeltaV End - Fix EvenHealing with Limbs.
+    }
 
-        var dspec = new DamageSpecifier();
+    private DamageSpecifier GetDamageSpec(IPrototypeManager protoMan, DamageableComponent damageable) // DeltaV - Fix EvenHealing with Limbs.
+    {
+        var damageSpecifier = new DamageSpecifier();    // DeltaV - Fix EvenHealing with Limbs.
```

Comment:
.

Reply:
What's the issue here? Want me to encapsulate the two lines with DeltaV start - end?

Reply:
Remove the extra spaces before the comment. There is always just one
```suggestion
        var damageSpecifier = new DamageSpecifier(); // DeltaV - Fix EvenHealing with Limbs.
```

Reply:
Oh, I usually use tab for spacing, since it's more pleasant to the eyes when the spacing of the comments is unified